docker/docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: "3.7"
+services:
+  pulsar:
+    image: apachepulsar/pulsar:2.5.0
+    command: bin/pulsar standalone
+    hostname: pulsar
+    ports:
+      - "8080:8080"
+      - "6650:6650"
+    restart: unless-stopped
+    volumes:
+      - "./data/:/pulsar/data"
+  dashboard:
+    image: apachepulsar/pulsar-manager:v0.1.0
+    ports:
+      - "9527:9527"
+    depends_on:
+      - pulsar
+    links:
+      - pulsar
+    volumes:
+      - "./data/:/data"
+    environment:
+      REDIRECT_HOST: "http://127.0.0.1"
+      REDIRECT_PORT: "9527"
+      DRIVER_CLASS_NAME: "org.postgresql.Driver"
```

Comment:
Do you have ever used HerdDB embedded in production  your environments ?

I think it can be a simpler way to get started and we are already using it in production in standalone mode with  quite big databases, several GB of data

Reply:
@eolivelli cool. we haven't yet. but that's a good. do you want to contribute a docker-compose setting that starts pulsar-manager using an embedded HerdDB?